docs/_config.yml

```diff
@@ -5,7 +5,7 @@ description: > # this means to ignore newlines until "baseurl:"
   Write an awesome description for your new site here. You can edit this
   line in _config.yml. It will appear in your document head meta (for
   Google search results) and in your feed.xml site description.
-baseurl: "/smarthome/" # the subpath of your site, e.g. /blog/
+baseurl: "/smarthome" # the subpath of your site, e.g. /blog/
```

Comment:
Maybe worth to remove the trailing slash from the "e.g. /blog/" as well? But then again, why don't we follow the example?

Reply:
> Maybe worth to remove the trailing slash from the "e.g. /blog/" as well?

That one comes from jekyll itself as an example. I wouldn't change it, but rather remove it completely.

> But then again, why don't we follow the example?

Because we would end up with 

```html
<a href="/smarthome//documentation/concepts/units-of-measurement.html">
```

No idea why though - Jekyll sometimes feels like pure magic.